ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/NtiUnderConsideration/Edit.cshtml.cs

```diff
@@ -69,11 +71,28 @@ public async Task<IActionResult> OnPostAsync()
 		{
 			try
 			{
+				
+				if (!ModelState.IsValid)
+				{
+					ResetOnValidationError();
+					ResetOnValidationError();
+					var data = PopulateNtiFromRequest();
+					var isChecked = data.NtiReasonsForConsidering.Where(c => c.Id != 0);
+					NTIReasonsToConsiderForUI = GetReasonsForUI(NtiModel).ToList();
```

Comment:
This function selects the selected item, so we don't need the logic below it on edit

Reply:
removed dup

Reply:
Passed in data isn't a radio button